corehq/apps/reports/dispatcher.py

```diff
@@ -297,12 +297,18 @@ def dispatch(self, request, *args, **kwargs):
     def permissions_check(self, report, request, domain=None, is_navigation_check=False):
         from corehq.motech.repeaters.views import DomainForwardingRepeatRecords
         from corehq.apps.export.views.incremental import IncrementalExportLogView
+        from corehq.apps.linked_domain.views import DomainLinkHistoryReport
+        from corehq.apps.linked_domain.util import can_access_linked_domains
+
         from corehq.toggles import INCREMENTAL_EXPORTS
 
         if (report.endswith(DomainForwardingRepeatRecords.__name__)
                 and not domain_has_privilege(domain, privileges.DATA_FORWARDING)):
             return False
-        if (report.endswith(IncrementalExportLogView.__name__) and not INCREMENTAL_EXPORTS.enabled(domain)):
+        if report.endswith(IncrementalExportLogView.__name__) and not INCREMENTAL_EXPORTS.enabled(domain):
+            return False
+        if (report.endswith(DomainLinkHistoryReport.__name__)
+                and not can_access_linked_domains(request.couch_user, domain)):
             return False
```

Comment:
@biyeun or @mjriley, I want to use this specific permission `can_access_linked_domains` for this report. The better solution seems to be adding a specific report dispatcher that inherits from `DomainReportDispatcher`, but this also appears to be the precedent. Curious to hear any thoughts.

Reply:
Offlined with Biyeun and decided a new report dispatcher to house release management features is the way to go.